backend/compact-connect/docs/design/README.md

```diff
@@ -31,6 +57,82 @@ efficient processing. A lambda receives messages from the SQS queue. Each messag
 ingested. The lambda receives the data and creates or updates a corresponding license record in the DynamoDB license
 data table.
 
-
 ### Asynchronous validation feedback
 Asynchronous validation feedback for boards to review is not yet implemented.
+
+## User Architecture
+[Back to top](#backend-design)
+
+Authentication with the CompactConnect backend will be controlled through Oauth2 via
+[AWS Cognito User Pools](https://github.com/csg-org/CompactConnect). Clients will be divided into two groups, each
+represented by an independent User Pool: [Staff Users](#staff-users) and [Licensee Users](#licensee-users). See
+the accompanying [architecture diagram](./users-arch-diagram.pdf) for an illustration.
+
+### Staff Users
+
+Staff users come with a variety of different permissions, depending on their role. There are Compact Executive
+Directors, Compact ED Staff, Board Executive Directors, Board ED Staff, and CSG Admins, each with different levels
+of ability to read and write data, and to administrate users. Read permissions are granted to a user for an entire
```

Comment:
Question: can a user have write permission without read permission? If not, is that something we're testing for?

Reply:
Yes, they can. In fact, when we create a machine-to-machine client, that may happen, depending on how boards end up wanting to use that capability.

Reply:
Got it!